toolkits/global/packages/global-forms/demo/context.json

```
@@ -0,0 +1,234 @@
+						"template": "text",
+						"label": "Your name (with error)",
+						"hint": "Your name is on your passport",
+						"error": "That is not a name, it’s a number",
+						"value": "987364",
+						"id": "your-name-error",
+						"name": "your-name-error"
+					},
+					{
+						"template": "text",
+						"label": "Your name (disabled)",
+						"id": "your-name-disabled",
+						"name": "your-name-disabled",
+						"disabled": true
+					}
+				]
+			},
+			{
+				"legend": "<h2>Selects</h2>",
+				"fields": [
+					{
+						"template": "select",
+						"label": "Animal (simple)",
+						"id": "select-animals",
+						"name": "select-animals",
+						"options": [
+							{
+								"label": "Horse",
+								"value": "Horse"
+							},
+							{
+								"label": "Monkey",
+								"value": "Monkey"
+							},
+							{
+								"label": "Platypus",
+								"value": "Platypus",
+								"selected": true
+							}
+						]
+					},
+					{
+						"template": "select",
+						"label": "Animal (with hint)",
+						"id": "select-animals-hint",
+						"name": "select-animals-hint",
+						"hint": "Which is your favorite?",
+						"options": [
+							{
+								"label": "Horse",
+								"value": "Horse"
+							},
+							{
+								"label": "Monkey",
+								"value": "Monkey"
+							},
+							{
+								"label": "Platypus",
+								"value": "Platypus",
+								"selected": true
+							}
+						]
+					}
+				]
+			},
+			{
+				"legend": "<h2>Radios</h2>",
+				"fields": [
+					{
+						"template": "radios",
+						"label": "Animal",
+						"id": "radios-animals",
+						"name": "radios-animals",
+						"inputs": [
+							{
+								"label": "Horse",
+								"value": "Horse",
+								"id": "radio-horse"
+							},
+							{
+								"label": "Monkey",
+								"value": "Monkey",
+								"id": "radio-monkey",
+								"fields": [
+									{
+										"template": "text",
+										"label": "What type of monkey?",
+										"id": "monkey-type",
+										"name": "monkey-type",
+										"hint": "Chimps are not monkeys",
+										"error": "Choose a monkey already!"
+									}
+								]
+							},
+							{
+								"label": "Platypus",
+								"value": "Platypus",
+								"id": "radio-platypus",
+								"checked": true
+							}
+						]
+					}
+				]
+			},
+			{
+				"legend": "<h2>Textareas</h2>",
+				"fields": [
+					{
+						"template": "textarea",
+						"label": "Enter some text",
+						"id": "textarea-example",
+						"name": "textarea-example",
+						"value": "This has been entered already",
+						"rows": "5"
+					}
+				]
+			},
+			{
+				"legend": "<h2>Checkboxes</h2>",
+				"fields": [
+					{
+						"template": "checkbox",
+						"label": "Agree to our terms (basic)",
+						"id": "checkbox-terms",
+						"name": "checkbox-terms"
+					},
+					{
+						"template": "checkbox",
+						"label": "Agree to our terms (with hint)",
+						"hint": "You should have read these. Read them, then agree to them.",
```

Changing to something I'd be happy to see in a real service

```suggestion
						"hint": "You must agree with the terms to continue your application",
```